### PR TITLE
Readme_jp updates

### PR DESCRIPTION
**Pull request by chack411**
# Documentation Update

## Change Description
I added a `README_jp.md`
I updated the `README.md`
I added the banner image of Shonan Code that was created by DALL-E.

## Checklist
- [x] I have read the code of conduct and the documentation guidelines
- [x] My documentation follows the project's writing style and standards
- [x] My documentation is accurate, clear, and complete
- [x] I have checked the links and references, and they are working correctly
- [x] I have tested the updated documentation and verified that it is helpful and informative

## Additional Notes
- Basically, I am changing the "paper" to a "self-introduction page". If you have any other good expressions, please update.
- 20,000 yen is a little expensive for a one-day event.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,67 @@
...
 * Cost: TBD (goal is under 20,000yen)
 
 At the heart of this event, we have a unique and engaging format:
 
-1. Engineers will submit papers showcasing their deep technical insights, concepts, and code.
-1. Selected papers will be presented by their authors during engaging and insightful sessions.
-1. Attendees will have the chance to share their papers in shorter "speed-dating" type sessions, fostering dynamic and rapid exchanges of ideas.
+1. Engineers will submit a self-introduction page (bio) showcasing their deep technical insights, concepts, code, and so on.
+1. Participants who submit engaging and insightful activities and knowledge will have the opportunity to present as speakers.
+1. Attendees will have the chance to share their technical knowledges in shorter "speed-dating" type sessions, fostering dynamic and rapid exchanges of ideas.
```
Comment (drewby) [inline README.md:19]:
```suggestion
1. Attendees will have the chance to share their technical knowledge in shorter "speed-dating" type sessions, fostering dynamic and rapid exchanges of ideas.
```

File: README.md [hunk 2]
```diff
@@ -1,60 +1,67 @@
...
 
 ## Shonan Area
-What sets this event apart is its stunning location in Hayama, Japan. With breathtaking views of Mount Fuji across Sagami Bay, you'll be inspired as you immerse yourself in the beauty of this seaside town. Enjoy a variety of local activities, including marine sports, mountain trails, shopping, and dining.
+What sets this event apart is its stunning location in Hayama, Japan. With breathtaking views of Mount Fuji across Sagami Bay, you'll be inspired as you immerse yourself in the beauty of this seaside town. Along with participating in the event, enjoy a variety of local activities, including marine sports, mountain trails, shopping, and dining.
 
 Don't miss this incredible opportunity to connect with like-minded engineers, learn from industry leaders, and contribute to shaping the future of software development and architecture. Share this invitation with fellow engineers and let's create the ultimate future together at the Shonan Code Summit (ShoCode)!
 
+<img src="resources/top_banner.jpg" alt="Shonan Code Summit" title="Shonan Code Summit" width="400" height="400">
```
Comment (drewby) [inline README.md:33]: LOVE THIS! :)

**Review by drewby — APPROVED** [review]
Suggested edits, but approved to merge.

File: README.md [hunk 3]
```diff
@@ -1,60 +1,67 @@
...
-This is your one-stop source for all event details, venue information, logistics, and costs. As an innovative technical event, we will be using GitHub to manage every aspect of the summit, including paper submissions and attendee registration.
+This is your one-stop source for all event details, venue information, logistics, and costs. As an innovative technical event, we will be using GitHub to manage every aspects of the summit, including the submission of your self-introduction (bio) page, which also doubles as registration.
 
 ## The Summit
 Join us for an unforgettable one-day event, the Shonan Code Summit (ShoCode), focused on exploring and exchanging cutting-edge ideas and knowledge in the world of software engineering and architecture. Our goal is to bring together the finest engineers who are passionate about creating a brighter future for developers and enhancing their professional fulfillment, and together, we will push the boundaries of innovation and design.
 
 * Location: TBD (Hayama, Japan)
-* Date: TBD (August/September) 
+* Date: TBD (August/September, 2023) 
 * Cost: TBD (goal is under 20,000yen)
```
Comment (drewby) [inline README.md:13]:
```suggestion
* Cost: TBD (goal is under 10,000yen)
```

File: README_jp.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+このリポジトリは、イベントのすべての詳細、会場情報、ロジスティクス、および参加費用のワンストップ情報源です。革新的な技術イベントとして、GitHub を使用して、参加登録を兼ねた自己紹介ページの提出など、サミットのあらゆる側面を管理します。
+
+## The Summit
+
+ソフトウェア エンジニアリングとアーキテクチャの世界における最先端のアイデアと知識を探求し、情報交換することに焦点を当てた忘れられない 1 Day イベント、Shonan Code Summit (ShoCode) にご参加ください。私たちの目標は、開発者のより明るい未来を創造し、専門的な充実感を高めることに情熱を傾ける最高のエンジニアを集めることであり、共にイノベーションとデザインの境界を押し広げます。
+
+* 場所: 未定 (葉山, 日本)
+* 日程: 未定 (2023 年 8 月 - 9 月を目標)
+* 参加費用: 未定 (¥20,000 以内を目標)
```
Comment (drewby) [inline README_jp.md:15]:
```suggestion
* 参加費用: 未定 (¥10,000 以内を目標)
```
